### ISSUE TITLE
Would you like to join in vue-admin and vue-bulma?

### ISSUE THREAD

**Issue by fundon** [fundon/vue-admin#92]

**Comment by fundon** [comment]
@luventa 有兴趣加入 https://github.com/fundon/vue-admin 和 https://github.com/vue-bulma 的开发吗？

**Comment by luventa** [comment]
非常有兴趣。
抱歉最近工作太忙，现在才回复你。
我也有几天没更新fork的代码了。

**Comment by luventa** [comment]
@fundon 请问你有其他联系方式吗？

**Comment by fundon** [comment]
抱歉，这两天在外旅游。加我微信 `-Fundon`。😊

**Comment by luventa** [comment]
@fundon 刚加了，但是好像不是你。
加我的吧，panyue1989

**Comment by GoldenIron** [comment]
After trying it out, i would love to join. Working on it is fun and i can see myself building application with it.

**Comment by fundon** [comment]
@GoldenIron Welcome!

**Comment by Thebigbignooby** [comment]
is this dead ?

**Comment by luventa** [comment]
@Thebigbignooby nope, still alive. Chinese new year is coming, we are too busy with daily work and housework.
